cmd/kritis/signer/main.go

```diff
@@ -172,18 +169,14 @@ func main() {
 				PublicKeys: []v1beta1.PublicKey{
 					{
 						KeyType:                  "PGP",
-						AsciiArmoredPgpPublicKey: base64.StdEncoding.EncodeToString([]byte(pubKey)),
+						AsciiArmoredPgpPublicKey: "",
```

Comment:
I suggest hiding this hack inside signer.go.  That way the signer API can be cleaned up a bit: it can receive a note name instead of the entire attestation authority.

Reply:
Done.